capplets/appearance/appearance-style.c

```diff
@@ -152,6 +153,31 @@ treeview_selection_changed_callback (GtkTreeSelection *selection, guint data)
 
     if (list_value) {
       g_settings_set_string (settings, key, list_value);
+
+    /*Load the gnome interface schema if we are running under wayland and it is present*/
+      if (!(GDK_IS_X11_DISPLAY (gdk_display_get_default())))
+      {
+        GSettingsSchemaSource *source = g_settings_schema_source_get_default ();
+
+        if (source)
+        {
+          GSettingsSchema *schema = g_settings_schema_source_lookup (source, INTERFACE_GNOME_SCHEMA, TRUE);
+          {
+            if (schema)
+            {
+              GSettings *interface_gnome_settings = NULL;
+              interface_gnome_settings = g_settings_new_full (schema, NULL, NULL);
+
+              /*Exclude marco theming under wayland, the compositor handles window border theming*/
+              if ((strcmp (key, MARCO_THEME_KEY) != 0) &&
+                 (strcmp (key,  WINDOW_TITLE_FONT_KEY) != 0) &&
+                 (strcmp (key,  WINDOW_TITLE_USES_SYSTEM_KEY) != 0))
```

Comment:
I'm not sure I'm a fan of excluding items instead of including them, but if you think it's better OK

Reply:
Intention here was basically to apply all the possible options save those we know are not used under wayland. Alternately we could include gtk theme, icon theme, cursor theme, cursor size if you feel this is a better approach

Reply:
https://github.com/mate-desktop/mate-control-center/compare/b29a2e863965cea9f161221835b652758c95f80b..fc7ec90d9a28031334ccb4c11f16a086e1ed232a

restores the logic of checking for desired conditions rather than excluding.

Reply:
Note that while we do have a cursor size key in GNOME, it does not seem to apply from here. I am about to head out on a major road trip so time will be limited

Reply:
Found the cursor-size issue: this is not applied from `treeview_selection_changed_callback()` at all, since this key is an integer and not a string. I will need to find out where that is changed, and apply it from there

Reply:
> restores the logic of checking for desired conditions rather than excluding.

I like that :+1: